docassemble/juvenilesealing/data/questions/petitioner-entrypoint-10.yml

```diff
@@ -38,16 +44,24 @@ code: |
     progressor.increment()  # 1
 
   progressor.increment()  # 2
-
+  show_docket_sheets_info = True
   needs_dockets = False
-  # If user doesn't answer yes for docket sheets, show user the docket sheet description
+  # If user doesn't answer yes for docket sheets,
```

Comment:
It's not clear to me what `answer yes for docket sheets` means. Is that yes to having docket sheets?
It's also not clear where the `show user why docket sheets are important` step happens

Reply:
I suggest "If the user doesn't have their certified copies already" for the first one, though I think the variable name might be documenting itself at this point (line 51).

For the second, how do we clarify that `wants_docket_sheets` (which is really "wants docket sheet instructions") is the "show them why it's important" step? `why_docket_sheets_are_important` isn't a useful variable name describing the reasons for the logic flow. It's only descriptive of the content of that screen.

Reply:
`have_docket_copies_already` seems sufficiently self-documenting to me. 

I think moving `# show users why docket sheets are important` to directly above the place where `wants_docket_instructions` is called would be fine. The distance between the comment and the code made me think they weren't related, especially since that piece of code already has its own set of comments